scale/job/models.py

```diff
@@ -1085,10 +1062,6 @@ def task_ended(self, job_exe_id, task, when, exit_code, stdout, stderr):
         :type when: :class:`datetime.datetime`
         :param exit_code: The task exit code, possibly None
         :type exit_code: int
-        :param stdout: The task stdout contents, possibly None
-        :type stdout: string
-        :param stderr: The task stderr contents, possibly None
-        :type stderr: string
         """
 
         # TODO: once we no longer have stdout and stderr in job execution model, transition this from selecting with
```

Comment:
Resolve this TODO by making task_ended() use an update query just like task_started() does. The locking was only needed when appending stdout/stderr to existing stdout/stderr contents which we are no longer doing. The update is faster and cleaner.